src/calculateRentalCost.test.js

```diff
@@ -3,21 +3,17 @@
 const calculateRentalCost = require('./calculateRentalCost');
 
 test(`Do not add any discount for less than 3 days`, () => {
-  expect(calculateRentalCost(2))
-    .toBe(80);
+  expect(calculateRentalCost(2)).toBe(80);
 });
 
 test('Should add the basic discount from 3 to 6 days of rent', () => {
-  expect(calculateRentalCost(3))
-    .toBe(100);
+  expect(calculateRentalCost(3)).toBe(100);
 });
 
 test('Should add the basic discount from 3 to 6 days of rent', () => {
-  expect(calculateRentalCost(6))
-    .toBe(220);
+  expect(calculateRentalCost(6)).toBe(220);
 });
 
 test('Should add an additional discount for 7 and more days of rent', () => {
-  expect(calculateRentalCost(7))
-    .toBe(230);
+  expect(calculateRentalCost(7)).toBe(230);
```

Comment:
Confirm that the expected value of 230 is correct for 7 days of rent. According to the task description, renting for 7 or more days should provide a $50 discount. If the daily rate is $40, then the expected cost for 7 days could be (7 * $40) - $50 = $230. If the implementation differs, this test case needs to be updated to match the correct expected value.